### PR TITLE
DD214 content for Discharge Wizard

### PR DESCRIPTION
**Pull request by bshyong**
Adds content for dd214 instructions

resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/6530

image

### CODE REVIEW

File: src/js/discharge-wizard/containers/GuidancePage.jsx [hunk 1]
```diff
@@ -16,6 +16,13 @@ class GuidancePage extends React.Component {
 
   componentDidMount() {
     window.scrollTo(0, 0);
+    localStorage.setItem('dw-viewed-guidance', true);
```
Comment (bshyong) [inline src/js/discharge-wizard/containers/GuidancePage.jsx]: all of the sessionStorage and localStorage stuff is for making sure we have desired behavior around people trying to navigate to certain pages directly, in the same window or in a new tab

File: src/js/discharge-wizard/components/FormQuestions.jsx [hunk 2]
```diff
@@ -81,7 +81,7 @@ class FormQuestions extends React.Component {
       { label: questionLabels[key]['7'], value: '7' },
     ];
 
-    const label = <h4>Which of the following <strong>best</strong> describes why you want to change your discharge paperwork? Choose the one that best applies to your situation. Note: If multiple options fit your situation, choose the answer that started the events leading to your discharge. (For example, if you experienced sexual assault and had PTSD resulting from that experience, choose sexual assault as your main reason.)</h4>;
+    const label = <h4>Which of the following <strong>best</strong> describes why you want to change your discharge paperwork? Choose the one that's closest to your situation. <strong>Note:</strong> If more than one of these fits your situation, choose the one that started the events leading to your discharge. For example, if you experienced sexual assault and have posttraumatic stress disorder (PTSD) resulting from that experience, choose sexual assault.</h4>;
```
Comment (U-DON) [inline src/js/discharge-wizard/components/FormQuestions.jsx]: It's "posttraumatic" here, but "post-traumatic" in the config below.

File: src/js/discharge-wizard/containers/GuidancePage.jsx [hunk 3]
```diff
@@ -16,6 +16,13 @@ class GuidancePage extends React.Component {
...
+
+    if (sessionStorage.getItem('dw-session-started')) {
+      sessionStorage.removeItem('dw-session-started');
```
Comment (U-DON) [inline src/js/discharge-wizard/containers/GuidancePage.jsx]: The sessionStorage key `dw-session-started` doesn't seem to be added anywhere in this change set. Not sure if it has been added in previous changes, but check to make sure.

**Review by U-DON — APPROVED** [review]
LGTM. Just noticed one inconsistency in copy and something with a sessionStorage key.

**Comment by amybehrens** [comment]
Should be "posttraumatic" (no hyphen)--earlier versions had the hyphen, which our copyeditor identified as incorrect per the Vets.gov style guide. 

Also, not sure what happened with the md file, but in the intro in the screen shot above, there should be a period after "BCNR" and the sentence after deleted ("do not send to the...").

**Pull request merged**